circle.yml

```diff
@@ -1,23 +1,23 @@
+general:
+  artifacts:
+    - "tmp"
+
 machine:
+  environment:
+    ACCESSLINT_MASTER_BRANCH: dev
   ruby:
     version:
       2.3.1
+  node:
+    version: 6.1.0
```

Comment:
@gemfarmer I don't recall at the moment what component failed with `0.10` (the default Circle CI version). I think it may have been promises, which are supported in `0.12`.